formatTest/unit_tests/expected_output/variants.re

```diff
@@ -400,32 +400,36 @@ let rec atLeastOneFlushableChildAndNoWipNoPending =
   | [] => false
   | [hd, ...tl] =>
     switch hd {
-    | OpaqueGraph {lifecycle: Reconciled(_, [])} =>
+    | OpaqueGraph({lifecycle: Reconciled(_, [])}) =>
       atLeastOneFlushableChildAndNoWipNoPending(
         tl,
         atPriority
       )
-    | OpaqueGraph {
-        lifecycle:
-          ReconciledFlushable(
-            priority,
-            _,
-            _,
-            _,
-            _,
-            _
-          )
-      }
-    | OpaqueGraph {
-        lifecycle:
-          NeverReconciledFlushable(
-            priority,
-            _,
-            _,
-            _,
-            _
-          )
-      }
+    | OpaqueGraph(
+        {
+          lifecycle:
+            ReconciledFlushable(
+              priority,
+              _,
+              _,
+              _,
+              _,
+              _
+            )
+        }
+      )
+    | OpaqueGraph(
+        {
+          lifecycle:
+            NeverReconciledFlushable(
+              priority,
+              _,
+              _,
+              _,
+              _
+            )
+        }
+      )
```

Comment:
Printing here could probably be improved in a number of ways. Would be worth testing something like:

```
| OG({
    ...
  })
```
and
```
| OG({
    ...
  }) when ...
```

This would consolidate extra lines and extra indentation.

Reply:
I'll take a look today.

Reply:
I'll merge this first then